### PR TITLE
Merge feature branch in main

### PR DESCRIPTION
**Pull request by GabrielChouinardLetourneau**
GitHub Issue: #

## Proposed Changes

 - [ ] Bug fix
 - [ ] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## Impact on version

- [ ] **Major** (Public API was modified.)
  - Public constructs (class, struct, delegate, enum, etc.) were removed or renamed.
  - Public members were removed or renamed.
  - Public method signatures were changed or renamed.
- [x] **Minor** (Public API was extended.)
  - Public constructs, members, or overloads were added.
- [ ] **Patch** (Public API was unchanged.)
  - A bug in behavior was fixed.
  - Documentation was changed.
- [ ] **None** (The library is unchanged.)
  - Only code under the `build` folder was changed.
  - Only code under the `.github` folder was changed.

## Checklist

Please check that your PR fulfills the following requirements:

- [ ] Documentation has been added/updated.
- [ ] Automated Unit / Integration tests for the changes have been added/updated.
- [ ] Updated [BREAKING_CHANGES.md](../BREAKING_CHANGES.md) (if you introduced a breaking change).
- [ ] Your conventional commits are aligned with the **Impact on version** section.

### CODE REVIEW

File: build/stage-build.yml [hunk 1]
```diff
@@ -15,6 +15,13 @@
   inputs:
     forceReinstallCredentialProvider: true
     
+- task: DotNetCoreCLI@2
+  displayName: 'Install dotnet 6 workloads'
+  inputs:
+    command: 'custom'
+    custom: 'workload'
+    arguments: 'install android ios'
```
Comment (carlh98) [inline build/stage-build.yml:23]: Add other workload

File: src/DynamicMvvm.Uno.WinUI/DynamicMvvm.Uno.WinUI.csproj [hunk 2]
```diff
@@ -0,0 +1,54 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
+	<PropertyGroup>
+		<TargetFrameworks>net6.0-windows10.0.18362;netstandard2.0;net6.0-android;net6.0-ios;</TargetFrameworks>
```
Comment (carlh98) [inline src/DynamicMvvm.Uno.WinUI/DynamicMvvm.Uno.WinUI.csproj:3]: add missing workload

**Review by carlh98 — CHANGES_REQUESTED** [review]

File: src/DynamicMvvm.Uno.WinUI/DynamicMvvm.Uno.WinUI.csproj [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+	<ItemGroup>
+		<None Include="..\..\README.md">
+			<Pack>True</Pack>
+			<PackagePath>\</PackagePath>
+		</None>
+	</ItemGroup>
+	
+	<ItemGroup>
+		<PackageReference Include="MSBuild.Sdk.Extras" Version="3.0.44" />
```
Comment (carlh98) [inline src/DynamicMvvm.Uno.WinUI/DynamicMvvm.Uno.WinUI.csproj:34]: you can remove msbuild package from here

File: build/stage-build.yml [hunk 4]
```diff
@@ -15,6 +15,13 @@
   inputs:
```
Comment (lamonfly) [inline build/stage-build.yml:15]: There shouldn't be a need to authenticate for NuGet packages.

**Review by jeanplevesque — APPROVED** [review]

**Review by carlh98 — APPROVED** [review]

**Pull request merged**